__tests__/app.test.js

```diff
@@ -42,7 +42,27 @@ describe('GET /api/topics', () => {
     });
   });
 
-  //STILL NEED TO DO TASK 4!
+  describe('GET /api/articles/:article_id', () => {
+    test('200: serves the correct article object when valid article_id provided', () => {
+      return request(app)
+      .get('/api/articles/2')
+      .expect(200)
+      .then(({ body }) => {
+          expect(body.article).toEqual(
+            expect.objectContaining({
+              author: expect.any(String),
+              title: expect.any(String),
+              article_id: expect.any(Number),
```

Comment:
Do we expect _any_ number for the id here? Asserting the article requested is returned would be good.